### PR TITLE
Add user service token store

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@gsvr ready for review

File: services/models.py [hunk 1]
```diff
@@ -44,3 +44,25 @@ class Status(models.Model):
...
     def __str__(self):  # __unicode__ on Python 2
         return str("Up status is %s at %s" % (self.up, self.created_at))
+
+
+@python_2_unicode_compatible
+class UserServiceToken(models.Model):
+
+    """
+    User Service Token model, seed-auth-api user tokens from seed services
+    user_id is consistent from auth-api
```
Comment (gsvr) [inline services/models.py]: I cannot figure out what this says

File: services/models.py [hunk 2]
```diff
@@ -44,3 +44,25 @@ class Status(models.Model):
...
+    """
+    service = models.ForeignKey(Service, related_name='service_users')
+    user_id = models.IntegerField(null=False, blank=False)
+    email = models.CharField(max_length=30, null=False, blank=False)
```
Comment (gsvr) [inline services/models.py:59]: Seems quite short for an email field

File: services/tasks.py [hunk 3]
```diff
@@ -109,3 +109,59 @@ def run(self, service_id, **kwargs):
                 exc_info=True)
 
 poll_service = PollService()
+
+
+class GetUserToken(Task):
+
+    """
+    Task to check the set the user tokens for a service
```
Comment (gsvr) [inline services/tasks.py]: typo

**Comment by gsvr** [comment]
Some remarks, :+1: when you've addressed them

File: services/models.py (hunk 2 above)
Comment (imsickofmaps) [inline services/models.py:59]: it's limited to the same as Django username email length

**Pull request merged**